content/doc/book/pipeline/jenkinsfile.adoc

```diff
@@ -944,10 +950,12 @@ Groovy string interpolation can inject rogue commands into command interpreters
 ======
 
 Another note of caution. Using Groovy string interpolation for user-controlled variables with steps that pass their arguments to command interpreters such as the `sh`, `bat`, `powershell`, or `pwsh` steps can result in problems analogous to SQL injection.
-This occurs when a user-controller variable (generally an environment variable, usually a parameter passed to the build) that contains special characters (e.g. `' " / \ $ & % ^ > < | ;`) is passed to the `sh`, `bat`, `powershell`, or `pwsh` steps using Groovy interpolation.
```

Comment:
the quotes broke the ascii doctor vscode plugin code highlighting in the file from this line on, I assume they weren't a critical part of the docs?
otherwise feel free to re-add them and we can live with the extension not being able to handle this case

Reply:
not critical at all 👍